app/config/config-default.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     foregroundColor: '#fff',
 
     // terminal background color
+    // opacity is only supported on macOS
```

Comment:
Maybe `transparency` is better. Or `Changing opacity`...

Reply:
I think this is good, it's called opacity everywhere i css.